apps/explorer/src/ui/utils/ButtonOrLink.tsx

```diff
@@ -1,21 +1,37 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { type ComponentProps, forwardRef } from 'react';
+import { type ComponentProps, forwardRef, useMemo } from 'react';
 
 import { LinkWithQuery, type LinkProps } from './LinkWithQuery';
 
 export interface ButtonOrLinkProps
     extends Omit<
         Partial<LinkProps> & ComponentProps<'a'> & ComponentProps<'button'>,
         'ref'
-    > {}
+    > {
+    prefixIcon?: JSX.Element;
```

Comment:
Two thing:
1. Can we rename these to just before and after? I also don't think we need `icon` in the prop name
2. Can we use ReactNode instead of jsx element? It's more correct.

Reply:
done